.github/workflows/release-package.yml

```
@@ -13,66 +13,81 @@
-          tag: ${{ steps.bump.outputs.new_tag }}
-          body: ${{ steps.generate_release_notes.outputs.body }}
-    outputs:
-      VersionNumber: ${{ steps.bump.outputs.new_tag }}
-    
-  publishpackage:
-    name: Publish Package
-    needs: 
-      - release
-    uses: ./.github/workflows/publish-package.yml
+      - restore_build_test
+    uses: './.github/workflows/tag-and-create-release.yml'
```

actionlint (1.7.7)

secret "GITHUB_NUGET_TOKEN" is required by "./.github/workflows/tag-and-create-release.yml" reusable workflow

(workflow-call)